summingbird-online/src/main/scala/com/twitter/summingbird/planner/OnlinePlan.scala

```diff
@@ -146,7 +159,7 @@ class OnlinePlan[P <: Platform[P], V](tail: Producer[P, V]) {
            * summer (to handle map-side aggregation). This check is here to prevent us from merging
```

Comment:
this comment is out of date now, right? You are not forcing a flatMap before a summer?